README.aspnet.md

```diff
@@ -0,0 +1,107 @@
+﻿# Featured Tags
+
+* `4.8`
+  * `docker pull mcr.microsoft.com/dotnet/framework/aspnet:4.8`
+* `3.5`
+  * `docker pull mcr.microsoft.com/dotnet/framework/aspnet:3.5`
+
+# What is ASP.NET
+
+ASP.NET is a high productivity framework for building Web Applications using Web Forms, MVC, Web API and SignalR.
+
+This repository contains `Dockerfile` definitions for ASP.NET Docker images. These images use the [IIS image](https://hub.docker.com/r/microsoft/iis/) as their base.
+
+This image contains:
+- Windows Server Core as the base OS
+- IIS 10 as Web Server
+- .NET Framework (multiple versions available)
+- .NET Extensibility for IIS
+
+# How to use these Images
+
+The [.NET Framework Docker samples](https://github.com/microsoft/dotnet-framework-docker/blob/master/samples/README.md) show various ways to use .NET Framework and Docker together.
+
+There is no need to specify an `ENTRYPOINT` in your Dockerfile since the `microsoft/aspnet` base image already includes an entrypoint application that monitors the status of the IIS World Wide Web Publishing Service (W3SVC).
+
+### Verify in the browser
+
+> With the current release, you can't use `http://localhost` to browse your site from the container host. This is because of a known behavior in WinNAT, and will be resolved in future. Until that is addressed, you need to use the IP address of the container.
+
+Once the container starts, you'll need to finds its IP address so that you can connect to your running container from a browser. You use the `docker inspect` command to do that:
```

Comment:
```suggestion
Once the container starts, you'll need to find its IP address so that you can connect to your running container from a browser. You use the `docker inspect` command to do that:
```